cmake/install/Findo3de.cmake.in

```diff
@@ -30,6 +30,16 @@ function(o3de_current_file_path path)
 endfunction()
 
 
+# If the generation of the SDK layout had LY_DISABLE_TEST_MODULES cache variable set, 
+# then the PAL_TRAIT_BUILD_TESTS_SUPPORTED value needs to be propagated as FALSE
+set(disable_test_modules @LY_DISABLE_TEST_MODULES@)
+if (disable_test_modules)
+    set_property(GLOBAL PROPERTY PAL_TRAIT_BUILD_TESTS_SUPPORTED_DEFAULT FALSE)
+else()
+    set_property(GLOBAL PROPERTY PAL_TRAIT_BUILD_TESTS_SUPPORTED_DEFAULT TRUE)
+endif()
```

Comment:
The code on line 49 below this is also trying to address this issue, should it be removed?

Reply:
Yeah this code can be moved there and setting of the PAL_TRAIT_BUILD_TESTS_SUPPORTED option to false doesn't below can be removed.

Reply:
whoops, i didn't see that little guy down there....